src/lib/theme/design-system/font-sizes.ts

```diff
@@ -1,17 +1,19 @@
 import { FontSizes } from '../theme.types';
 
 export const fontSizes: Record<FontSizes, string> = {
-  xs: '0.75rem',
-  s: '0.875rem',
-  m: '1rem',
-  l: '1.125rem',
-  xl: '1.25rem',
-  '2xl': '1.5rem',
-  '3xl': '1.75rem',
-  '4xl': '2rem',
-  '5xl': '2.25rem',
-  '6xl': '2.5rem',
-  '7xl': '2.75rem',
-  '8xl': '3rem',
-  '9xl': '3.5rem',
+  labelSmall: '0.688rem',
```

Comment:
keep the name logic, to avoid same values with diff keys

Reply:
Got it